### ISSUE TITLE
Serving cms content from a separate database does not work and is useless for development

### ISSUE THREAD

**Issue by rubyconvict** [comfy/comfortable-mexican-sofa#425]
Rails 4: it does not work at all, also there is a better way, read [using-activerecord-with-multiple-databases](http://pivotallabs.com/using-activerecord-with-multiple-databases/), unfortunately, we have to override all cms models to inherit base class, and there is no dynamic way to do it, also we need to remove Cms::Base inheritance.

Good thing is that now it works, but so what? Can we develop with it - no. Maybe it is good for existing, undeveloped, legacy databases...

Funny thing is that if we did, now we cannot override sofa's own cms migrations, adding our own version, we end up with:
`ActiveRecord::Base.establish_connection "#{ComfortableMexicanSofa.config.database_config}_#{Rails.env}"`
`==  CustomCreateCms: migrating ================================================`
`-- create_table(:cms_sites)`
`rake aborted!`
`An error has occurred, all later migrations canceled:`
`undefined method`escape' for nil:NilClass/home/mkalita/4jobs/vendor/bundle/ruby/2.0.0/gems/activerecord-4.0.2/lib/active_record/connection_adapters/mysql2_adapter.rb:77:in `quote_string'`

In conclusion, wiki should have appropriate info.
https://github.com/comfy/comfortable-mexican-sofa/wiki/Serving-cms-content-from-a-separate-database
